packages/integrations-hcl/strategies/nomad-pack/metadata.hcl.ts

```diff
@@ -0,0 +1,23 @@
+import { z } from 'zod'
+
+const app = z.object({
+  url: z.string(),
+})
+
+const pack = z.object({
+  name: z.string(),
+  description: z.string(),
+})
+
+const integration = z.object({
+  identifier: z.string(),
+  flags: z.string().array().default([]),
+})
+
+const schema = z.object({
+  app: app.array().length(1),
+  pack: pack.array().length(1),
+  integration: integration.array().length(1),
+})
```

Comment:
Looking at https://github.com/hashicorp/nomad-pack-community-registry/blob/main/packs/chaotic_ngine/metadata.hcl

Are `app` and `pack` Nomad-owned stanzas? and `integration` Digital-owned? It might be useful to note that here.

Also, we might want to add the missing `app` and `pack` fields here, even though we may not necessarily need them.

Reply:
> Are app and pack Nomad-owned stanzas? and integration Digital-owned? It might be useful to note that here.

Yes this is the right understanding.  I will note this!

> Also, we might want to add the missing app and pack fields here, even though we may not necessarily need them.

I would be concerned that we would be over-validating.  For example, if they have a field _xyz_ which is currently required, I wouldn't want to specify it as such in our configuration if we don't need to use it.  In the event that the Nomad team wants to change their specification, I wouldn't want to have to change anything on our end (unless it was fields that we were touching).